#163388100 Add tests for update article reducer and actions

PaulinhoNdegwa opened the pull request:
### What does this PR do?
- Add tests for update article reducer and actions
- Improve styling of the site

### Description of Task to be completed?
-  Test update article reducer and actions
- Improve styling of the site

### How should this be manually tested?

- Clone the github repository
`git clone https://github.com/andela/ah-the-phoenix-frontend.git`
- Change your working directory to the project folder
`cd ah-fulldeck-frontend`
- Checkout to thebranch ft-user-profile-163388099
`git checkout ft-user-login-163388097`
- Install the required dependencies using
` npm install`
- Run tests
` ./node_modules/.bin/jest`

### Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#163388100

ezkemboi committed:
Merge pull request #28 from andela/ft-rate-article-163388101

#163388101 User should be able to rate an article

ezrqnkemboi commented on src/views/Signup/Signup.js:
```diff
@@ -7,6 +7,8 @@ const Signup = props => {
   return (
     <div className="auth-form">
       <Auth authaction='signup' history={props.history} />
+      <br></br>
```
use `<br />`

ezrqnkemboi commented on src/views/Signup/Signup.js:
```diff
@@ -7,6 +7,8 @@ const Signup = props => {
     <div className="auth-form">
       <Auth authaction='signup' history={props.history} />
+      <br></br>
+      <br></br>
```
use `<br />`

ezrqnkemboi commented on src/views/Login/Login.js:
```diff
@@ -22,6 +22,8 @@ export class Login extends Component {
     return (
       <div className="auth-form">
         <AuthForm authaction='login' />
+        <br></br>
```
use `<br />`

ezrqnkemboi commented on src/redux/actioncreators/updateArticle.js:
```diff
@@ -20,6 +19,7 @@ export const updateArticle = (article_slug, payload) => dispatch => {
     dispatch(updateArticleRequest())
     axiosWithToken.patch("api/v1/articles/" + article_slug + "/", payload)
         .then(res => {
+            console.log(res.data)
```
You can remove this console log.

PaulinhoNdegwa committed:
feat(article): Add tests for update article

- Write tests for update actions and reducer
- Improve CSS styling

[Delivers #163388100]
